force-app/main/default/layouts/Tv_Serie__c-Custom Tv Serie Layout.layout-meta.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
add also layouts for other objects (standard layouts if you didn't create custom)